Engine/Setup/BaseSetupHandler.cs

```diff
@@ -152,5 +153,23 @@ public static void LoadBacktestJobAccountCurrency(IAlgorithm algorithm, Backtest
                 algorithm.SetAccountCurrency(job.CashAmount.Value.Currency);
             }
         }
+
+        /// <summary>
+        /// Get the available data feeds from config.json,
+        /// If none available, throw an error
+        /// </summary>
+        public static Dictionary<SecurityType, List<TickType>> GetConfiguredDataFeeds()
+        {
+            var dataFeedsConfigString = Config.Get("security-data-feeds");
+
+            if (!dataFeedsConfigString.IsNullOrEmpty())
+            {
+                Dictionary<SecurityType, List<TickType>> dataFeeds = new Dictionary<SecurityType, List<TickType>>();
```

Comment:
nit - don't think this line is required? The `new` resulting object will be overriden bellow

Reply:
Missed that in the refactor when I let it pass null back, ty 😁